src/main/java/org/lsc/service/SyncReplSourceService.java

```diff
@@ -244,7 +247,10 @@ public IBean getBean(final String id, final LscDatasets pivotAttrs, boolean from
             // When launching getBean in clean phase, the search base should be the Base DN, not the entry ID
             String searchBaseDn = (fromSameService ? id : baseDn);
             SearchScope searchScope = (fromSameService ? SearchScope.OBJECT : SearchScope.SUBTREE);
-			if(getAttrs() != null) {
+			if (!connection.isConnected()) {
+				connection = getConnection(ldapConn);
+			}
```

Comment:
indentation issue?

Reply:
Not that I can see in Github or my IDE. There is a lot of code in LSC indented with spaces instead of tabs, it could lead to format hazard in github diffs. I use tabs only.

Reply:
in github, the (if (!connection...) starts at the same level than the old (if(getAttrs()...).
The new if(getAttrs...) (see below) is not at the same level of indentation. The new level seems better, as it is aligned with SearchScope searchscope...

Reply:
see this view for example: https://github.com/lsc-project/lsc/pull/42/files

Reply:
I fixed the whole class indentation (replace all dots indents by tabs), https://github.com/lsc-project/lsc/blob/7c062813a40f485c3e92de8e79a81f748af0c15b/src/main/java/org/lsc/service/SyncReplSourceService.java